src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -88,12 +88,12 @@ public override bool Execute()
                         blobNames = blobNames.Where(b => b.StartsWith(BlobNamePrefix)).ToList<string>();
                     }
                 }
-                
```

Comment:
I'm not sure what these white space changes are, but we should revert them to clean up the diff a bit. (There is also a white space change in the LogWarning string on line 176 that we should also revert.)

Reply:
Ahh, I see what happened here. These were lines that had trailing whitespaces in them so it seems Visual studio's code formatter cleaned them up. The LN176 was a manual change I made to remove an extra whitespace after a period (there were two, and I guess it triggered me 😛). I can revert them, but I think at least the whitespace on empty lines is worth cleaning up eventually as it may show up again every time someone runs some kind of code formatter.

Reply:
I don't really mind these changes, but I do usually go through my diffs locally to make sure there aren't pure formatting changes like these that could cause merge conflicts down the road.

Reply:
I reverted the extra space in Line 176 as that's technically a change in the warning message. I kept the changes that remove the whitespace on empty lines as they were inconsistent with the rest of the file. Do let me know if you believe strongly and I will revert those as well.